tests/legacy/back-compatible-versions.txt

```diff
@@ -70,3 +70,4 @@
 1.7.5
 1.7.6
 1.7.7
+1.8.0
```

Comment:
Somehow the change here doesn't trigger the PyTorch full tests. Looking into this

Reply:
I think it does not trigger any GH action, right?
In such case it is expected as GH can't trigger any other GH action...

Reply:
It doesn't at the moment, but it should because this is the list of the test cases that pytest runs.